doc/manuals/user/initial_notification.md

```diff
@@ -121,12 +121,15 @@ initial notification is required.
 Between both options, We have opted to implement the more flexible approach, which provide 
 the initial notification to users who can exploit is and can be ignore for who doesn't need it.
 
+For ignoring the initial notification, a URI parameter `options=skipInitialNotification can` be 
+used. Using this URI parameter, initial notification can be skipped at Subscription 
+creation/updation time. 
+
 ## Additional considerations
 
 Note that initial notification is not always sent. It is sent only in the case some entity 
-matches the subscription criteria at subscription creation/update time. But if the matching
-occurs, it is not possible at the present moment to avoid it. Such possibility has been 
-proposed and it is [part of the present backlog](https://github.com/telefonicaid/fiware-orion/issues/920).
+matches the subscription criteria at subscription creation/update time. If the matching occurs, 
+it is possible to avoid it by using URI parameter.
```

Comment:
`..by using URI option (check [above section](# Avoid initial notification)).`

Reply:
fixed in 6969514